layouts/release/section.html

```diff
@@ -1,33 +1,215 @@
+            {{ $normalized_cta_url := .Site.Params.call_for_action_url }}
+            {{ if (eq (len (findRE `(\.[^\/]{3,4}|\/)$` $normalized_cta_url)) 0) }}
+              {{ $normalized_cta_url = printf "%s/" $normalized_cta_url }}
+            {{ end }}
+            
+            {{ $abs_cfa := printf "%s" ($normalized_cta_url | absURL) }}
+            {{ $is_cfa_page := 0 }}
+            {{ if or (eq $normalized_cta_url .Permalink) (eq $abs_cfa .Permalink) }}
+              {{ $is_cfa_page = 1 }}
+            {{ end }}
+            {{ if and (eq $has_cfa 1) (eq .Page.Params.show_call_for_action_button "true") }}
+              <div class="{{ $.Site.Params.call_for_action_wrapper_classes | default "float-right hidden-xs" }}" id="btn-call-for-action">
+                <div id="btn-call-for-action"><a href="{{ .Site.Params.call_for_action_url }}" class="{{ .Site.Params.call_for_action_classes | default "btn btn-huge btn-warning" }}"><i class="fa {{ .Site.Params.call_for_action_icon }}"></i> {{ .Site.Params.call_for_action_text }}</a></div>
+              </div>
+            {{ end }}
+            <div class="{{ $.Site.Params.main_menu_classes | default "navbar float-sm-right yamm" }}" id="main-menu">
+              <div class="navbar-collapse collapse reset float-right" id="navbar-main-menu">
+                <ul class="nav navbar-nav">
+                  {{ partial "nav.html" . }}
+                </ul>
+              </div>
+              {{ partial "nav_toggle.html" . }}
+            </div>
           </div>
         </div>
       </div>
     </div>
+    <!---Navbar end --->
+
+    {{ if isset .Page.Params "headline" }}
+    <div class="jumbotron featured-jumbotron featured-jumbotron-download vertical-align flex-column margin-bottom-0">
+
+      <div class="flex-grow-container vertical-align flex-wrap">
+        <div class="col-md-10 flex-column">
+          <p style="align-self: flex-start;"><span class="p text-primary">version 9</span> new release</p>
+          <h1>Jakarta&nbsp;EE</h1>
+        </div>
+        <div class="col-md-2 col-md-offset-2 vertical-align">
+          <span class="jumbotron-right-content"><i>9</i></span>
+        </div>
+        <div class="col-md-9 col-md-offset-1 vertical-align" style="text-align: left;">
+          <p><span class="p text-primary">An open, community-driven platform</span> <br> for future innovation of <br> enterprise Java technologies</p>
+        </div>
+      </div>
+
+      <div class="container jumbotron-button-container">
+        <ul class="list-inline buttons">
+          <li class="vcenter"><a class="btn btn-primary vertical-align" href="#">DOWNLOAD <br/> COMPATIBLE PRODUCTS</a></li>
+          <li class="vcenter"><a class="btn btn-primary vertical-align" href="#">SPECIFICATIONS</a></li>
+        </ul>
+      </div>
+    </div>
     {{ end }}
   </header>
+{{ end }}
+
+{{ define "footer" }}
```

Comment:
Are we not removing this?

Reply:
https://github.com/EclipseFdn/hugo-solstice-theme/pull/179 have not merged yet 😂